internal/plugins/ansible/templates/roles/files_dir.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package roles
+
+import (
+	"path/filepath"
+
+	"sigs.k8s.io/kubebuilder/pkg/model/file"
+
+	"github.com/operator-framework/operator-sdk/internal/plugins/ansible/constants"
+)
+
+const filesPath = "files" + constants.FilePathSep + ".placeholder"
+
+type RoleFiles struct {
+	file.TemplateMixin
+	file.ResourceMixin
+}
+
+// SetTemplateDefaults implements input.Template
+func (f *RoleFiles) SetTemplateDefaults() error {
+	if f.Path == "" {
+		f.Path = filepath.Join(constants.RolesDir, "%[kind]", filesPath)
+		f.Path = f.Resource.Replacer().Replace(f.Path)
+	}
```

Comment:
```suggestion
	if f.Path == "" {
		f.Path = filepath.Join("roles", "%[kind]", filesPath)
	}
	f.Path = f.Resource.Replacer().Replace(f.Path)
```


IMO has no need for the const. However, if we would like to centralize it then it should be done to follow the current desing by adding an const in the init/api and passing it as arg. see the values as KustomizeVersion for example.